pyface/base_toolkit.py

```diff
@@ -68,11 +68,14 @@
 import sys
 
 try:
+    import importlib_metadata
```

Comment:
I'd prefer something that defaults to the std. lib. functionality where it's available and only uses `importlib_metadata` as a fallback, since the std. lib. is really what we _want_ to be using: we don't want to be relying on the 3rd party version for any longer than we have to.

With the changes below, we don't actually need to do this reversal, right?

Reply:
with these changes, we don't need this reversal but I still chose the reversal specifically we made `importlib_metadata` a required dependency for Python < 3.10 - https://github.com/enthought/pyface/blob/91c93aef0ad034d9060c473457a6ac36514935fd/pyface/__init__.py#L22

Reply:
Understood, but I'd prefer to get rid of the `importlib_metadata` dependency as soon as we can - i.e., with Python 3.8.